www/_posts/2017-10-30-fetch-release.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+author:
+    name: Audrey So
+    url: https://twitter.com/aud_rey_so
+title:  "Cordova-Fetch@1.2.1 Released!"
+date: 2017-10-30
+categories: news
+tags: release tools cordova-fetch
+---
+
+We just released some changes to `cordova-fetch`!
+
+* [cordova-fetch@1.2.1](https://www.npmjs.com/package/cordova-fetch)
+
+Release Highlights:
+* Support added for node 8.
+
+<!--more-->
+# Changes include:
+
+## cordova-fetch
+
+* [CB-13504](https://issues.apache.org/jira/browse/CB-13504) : Updated `package.json` versions for `cordova-fetch`@1.2.1 release.
+* [CB-13501](https://issues.apache.org/jira/browse/CB-13501) : Added support for node 8.
+* [CB-13492](https://issues.apache.org/jira/browse/CB-13492) : Updated opts.save and included tests for no-save.
+* [CB-13380](https://issues.apache.org/jira/browse/CB-13380) : Incremented package version to -dev.
```

Comment:
I would delete line 27. It is an automated commit from coho we usually delete out during blog posts